composer.json

```diff
@@ -3,7 +3,7 @@
   "description": "PrestaShop offers a free, fully scalable, Open Source e-commerce solution.",
   "type": "project",
   "require": {
-    "php": ">=7.1.3",
+    "php": "^7.1.3",
```

Comment:
I don't think this change is right. This should support PHP versions higher than `7.1.3`.

Reply:
This is exactly what it does, it allows >= `7.1.3` < `8.0`

Reply:
Thanks for the explanation, caret symbol's meaning in PHP is different than in other languages.